### DISCUSSION TITLE
Attaching Documents But Not Near Campus

### DISCUSSION THREAD

**Discussion by OchiengEmmanuel** [community/community#139536] [GitHub Education]
### Hi everyone,

Hi all,
I am an online student having difficulty creating a GitHub Education account. I have tried sharing the documents I have but keep failing each time I try. I keep getting this message . There is no option to upload these documents as it only gives me an option to take pictures from the front, be it via phone, or my computer.
<img width="395" alt="image" src="https://github.com/user-attachments/assets/67ea35ef-d0b7-4933-afb3-1f3e07752bb1">

**Comment by VadymMakohon** [comment]
Hi @OchiengEmmanuel, I suggest reaching out to GitHub support by opening a ticket for assistance with this issue. I went through the process of applying for the GitHub Education Program, but my submission was denied. If your images are unclear, in another language, or missing key details, your application might be rejected. Ensure that your photos are clear, meet the guidelines, and if they are not in English, include a translated version before trying again.

If this response was helpful, I’d appreciate it if you could mark it as the answer. Thanks!
